home.html

```diff
@@ -0,0 +1,269 @@
+        </div>
+        <p class="head-para">
+            A joyful celebration believing in value of openess and sharing, creating
+    a positive change with people from over 80 countries is taking place in October in Korea.
+        </p>
+            <p class="desk-home">
+                <span>2015.10.15(THU)~16(FRI)</span><br>
+                @Nation Museuum of Korea, Art centre Nabi and more
+            </p>
+
+            </section>
+          </div>
+        </div>
+            <section class="">
+              
+              <div class="container">
+                <div class="prog1">
+                  <h2 >Main Program</h2>
+                  <hr class="line">
+                </div>
+              
+              <div class="flex">
+                <div class="card">
+                  <img class="card-image" src="web-cons/1.png" alt="">
+                  <h3 class="title">Lecture</h3>
+                  <p class="card-para">
+                    Listen to the speaker from  various countries about the message of sharing and  opening.
+                  </p>
+                </div>
+            
+                <div class="card">
+                  <img class="card-image" src="web-cons/2.png" alt="">
+                  <h3 class="title">CC Exhibition</h3>
+                  <p class="card-para">
+                    appreciate varrious creations appying CC license of art of artists, organized from Art center nabi
+                  </p>
+                  </p>
+                </div>
+            
+                <div class="card">
+                  <img class="card-image" src="web-cons/3.png" alt="">
+                  <h3 class="title">Forum</h3>
+                  <p class="card-para">
+                   Have time to talk share your though and opinion with experts for each topic.
+                  </p>
+                  </p>
+                </div>
+            
+                <div class="card">
+                  <img class="card-image" src="web-cons/4.png" alt="">
+                  <h3 class="title">WorkShop</h3>
+                  <p class="card-para">
+                    Try creating your own work using open source license rather than just watching 
+                  </p>
+                  </p>
+                </div>
+            
+                <div class="card">
+                  <img class="card-image" src="web-cons/5.png" alt="">
+                  <h3 class="title">CC Ignite</h3>
+                  <p class="card-para">
+                    Get opportunities to network woth CC affilates around the world, also atfer the summit
+                  </p>
+                  </p>
+                </div>
+               
+            </div>
+            <button id="btn" class="join-btn">Join CC Global Summit 2015</button><br>
+
+            <p class="prog"> see the whole program</p>
+          </div>
+        </div>
+            <div>
+            </section>          
+<section>
```

Comment:
* Kindly ensure The section "Featured speakers" should be created dynamically in JavaScript.
You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically. [Check requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=The%20section%20%22Featured,this%20section%20dynamically.)